lib/engine/game/g_1812/step/loan_operations.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/base'
+require_relative '../../g_1867/step/loan_operations'
+
+module Engine
+  module Game
+    module G1812
+      module Step
+        class LoanOperations < G1867::Step::LoanOperations
+          def unpaid_interest(entity, owed)
+            @log << "#{entity.name} owes #{format_currency(owed)} in loan interest but has #{format_currency(entity.cash)}"
+            @log << "#{entity.name} will pay all remaining treasury cash (#{format_currency(entity.cash)}) to the "\
```

Comment:
It needs to be `@game.format_currency` here, doesn't it?

Reply:
fixed